spec/support/language_helpers.rb

```diff
@@ -1,24 +0,0 @@
-# typed: false
-
-module LanguageHelpers
-  module HashHelpers
-    # Introduced in Ruby 2.5
-    def transform_keys
-      result = self.class.new
-      each_key do |key|
-        result[yield(key)] = self[key]
-      end
-      result
-    end
-
-    def stringify_keys
-      transform_keys(&:to_s)
-    end
-
-    def symbolize_keys
-      transform_keys(&:to_sym)
-    end
-  end
-end
-
-Hash.prepend(LanguageHelpers::HashHelpers)
```

Comment:
This is actually quite dangerous, as it masks the fact that Ruby < 2.5 does not have these methods.